packages/mobile/src/components/top-tab-bar/TopTabBar.tsx

```diff
@@ -115,43 +125,47 @@ export const TopTabBar = ({ state, descriptors, navigation, position }) => {
 
   return (
     <View style={styles.tabBarContainer}>
-      {state.routes.map((route, index) => {
-        const { options } = descriptors[route.key]
-        const label = options.tabBarLabel ?? options.title ?? route.name
-        const icon = options.tabBarIcon({ color: neutral })
-
-        const inputRange = state.routes.map((_, i) => i)
-        const opacity = position.interpolate({
-          inputRange,
-          outputRange: inputRange.map(i => (i === index ? 1 : 0.52)) // opacity range
-        })
-
-        return (
-          <View key={label} style={styles.tabContainer}>
-            <TouchableOpacity
-              accessibilityLabel={options.tabBarAccessibilityLabel}
-              accessibilityRole='button'
-              accessibilityState={{ selected: isFocused(index) }}
-              activeOpacity={0.8}
-              onLongPress={() => onLongPress(route, index)}
-              onPress={() => onPress(route, index)}
-              style={styles.tab}
-              testID={options.tabBarTestID}
-            >
-              <Animated.View style={{ opacity }}>{icon}</Animated.View>
-              <Animated.Text style={[styles.tabText, { opacity }]}>
-                {label}
-              </Animated.Text>
-            </TouchableOpacity>
-          </View>
-        )
-      })}
+      <View
```

Comment:
just wondering if we can combine the styles with tabBarContainer and tabsContainer and just have one View?

Reply:
No because the outer one contains the tab indicator and setting the background color of that would include the background of the indicator

Reply:
ah word, thanks for clarifying